api/src/test/java/org/openmrs/module/pihcore/event/PatientEventConsumerTest.java

```diff
@@ -0,0 +1,40 @@
+package org.openmrs.module.pihcore.event;
+
+import org.junit.jupiter.api.Test;
+
+public class PatientEventConsumerTest {
+
+    @Test
+    public void shouldTrackPatientChanges() throws Exception {
+        /*
```

Comment:
This was working when I wrote this in the dbevent module, I'll see if it can get it working in pihcore, but I may not want to do so.  The use of MySQL test containers is a bit complicated with running our builds in CI in docker itself...